corehq/apps/ota/decorators.py

```diff
@@ -79,7 +79,7 @@ def mobile_auth_or_formplayer(view_func):
     This decorator is used only for anonymous web apps and SMS forms.
     Endpoints with this decorator will not enforce two factor authentication.
     """
-    return get_multi_auth_decorator(default=BASIC, allow_formplayer=True)(
+    return get_multi_auth_decorator(default=BASIC, allow_formplayer=True, oauth_scopes=['sync'])(
```

Comment:
I think the scope should also be added to the list in settings: `OAUTH2_PROVIDER`

Reply:
I did add that to the list. Unless I am missing another place? https://github.com/dimagi/commcare-hq/pull/33300/files#diff-dc2c79b556e950acc7dea90175fd5febba22bf9fa4faaba1b61c3b3d5acdc9e9R904

Reply:
Oh weird, I didn't see that. Thanks.